src/shout_subgroup/database.py

```diff
@@ -0,0 +1,32 @@
+# Initialize the file-based SQLite database
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from shout_subgroup.models import Base, UserModel, SubgroupModel, GroupChatModel
+
+engine = create_engine('sqlite:///pingdem_database.db', echo=True)
```

Comment:
This is a file database instead of a in-memory one. Chose this b/c we can use tools to inspect the database.